cmd/qri.go

```diff
@@ -213,7 +213,11 @@ func (o *QriOptions) RegistryRequests() (*lib.RegistryRequests, error) {
 	if err := o.init(); err != nil {
 		return nil, err
 	}
-	return lib.NewRegistryRequests(o.repo, o.rpc), nil
+	req := lib.NewRegistryRequests(o.repo, o.rpc)
+	if o.node != nil {
```

Comment:
Instead of this, copy what is going on with lib/datasets.go, where there's NewDatasetRequestsWithNode, which takes a third parameter for the *QriNode. Doesn't matter if you pass a null pointer, there's the default anyway.